### PR TITLE
[systemtest] Remove duplicated methods vol.2

### PR DESCRIPTION
**Pull request by im-konge**
Signed-off-by: Lukas Kral <lkral@redhat.com>

### Type of change

- Enhancement

### Description

This PR will remove single used methods and duplicated methods from `StrimziUpgradeST`

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [x] Remove duplicated methods
- [x] Remove single used methods
- [ ] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java [hunk 1]
```diff
@@ -317,9 +317,17 @@ private void performUpgrade(JsonObject testParameters, int produceMessagesCount,
...
+
         LOGGER.info("Waiting for CO upgrade");
         DeploymentUtils.waitForDeploymentReady("strimzi-cluster-operator", 1);
-        waitForRollingUpdate();
+        LOGGER.info("Waiting for ZK StatefulSet roll");
+        StatefulSetUtils.waitTillSsHasRolled(zkStsName, 3, zkPods);
+        LOGGER.info("Waiting for Kafka StatefulSet roll");
+        StatefulSetUtils.waitTillSsHasRolled(kafkaStsName, 3, kafkaPods);
+        LOGGER.info("Waiting for EO Deployment roll");
+
```
Comment (Frawless) [inline systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java]: Remove it

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=StrimziUpgradeST

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=io.strimzi.systemtest.upgrade.StrimziUpgradeST

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=io.strimzi.systemtest.upgrade.StrimziUpgradeST

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=upgrade testcase=StrimziUpgradeST

**Comment by im-konge** [comment]
these tests are known issues, we are working fixes

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=upgrade testcase=StrimziUpgradeST

**Comment by im-konge** [comment]
It was false-negative, I wanted to try that this way ... ready to merge

**Comment by im-konge** [comment]
Travis is green.

**Pull request merged**